library/ziti.c

```diff
@@ -1128,6 +1128,7 @@ static void session_post_auth_query_cb(ziti_context ztx, int status, void *ctx)
         if (!ztx->no_current_edge_routers) {
             ziti_ctrl_current_edge_routers(&ztx->controller, edge_routers_cb, ztx);
         }
+        
```

Comment:
This was tested by adding the following function above:

```
void debug_timer(uv_timer_t *t){
    ziti_context ztx = t->data;

    ziti_re_auth(ztx);

    FREE(t);
}
```


and starting a timer here

```
        NEWP(timer, uv_timer_t);
        uv_timer_init(ztx->loop, timer);
        timer->data = ztx;
        uv_timer_start(timer, debug_timer, 15000, 0)
```